src/global/globalNode.js

```diff
@@ -19,14 +19,15 @@ import Node from '../base/node';
 import FocusedNodeView from '../focused/focusedNodeView';
 
 class GlobalNode extends Node {
-  constructor (node) {
+  constructor (node, entryNode) {
     node.size = node.size || 120;
-    super(node, 'global');
+    super(node, 'global', entryNode);
     this.refreshLoaded();
   }
 
   isInteractive () {
-    return !this.isEntryNode();
+    return true;
```

Comment:
True, those should probably be separate. Maybe a `isDraggable()` should be what can be dragged, which can default to true. Then `isInteractive()` can be strictly reserved for the node being selectable. Which reminds me that we can't just have it be if the node has children, because it's used for highlighting and selecting too.